src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -713,22 +713,18 @@ symvmFromCommand :: Command Options.Unwrapped -> Query EVM.VM
 symvmFromCommand cmd = do
   caller' <- maybe (SAddr <$> freshVar_) (return . litAddr) (caller cmd)
   callvalue' <- maybe (sw256 <$> freshVar_) (return . w256lit) (value cmd)
-  (calldata', cdlen, pathCond) <- case (calldata cmd, sig cmd) of
-    -- fully abstract calldata (up to 1024 bytes)
+  calldata' <- case (calldata cmd, sig cmd) of
```

Comment:
If I'm reading this right, there is currently no command line flag that allows the user to execute with calldata set to a `DynamicSymBuffer`?

Reply:
Right now that's true. You can with `loadSymVM` though, which I've been experimenting with. Will include it in the cli once things mature